FirstCeo/FirstCeo/Page/CustomView.swift

```diff
@@ -52,7 +52,9 @@ class CustomView: UIView {
         backgroundColor = .white
         layer.cornerRadius = 15
         clipsToBounds = true
-        setHeight(height: 175)
+        setHeight(height: 165)
```

Comment:
setHeight의 경우 heght 라벨이 중복으로 나오기 때문에 다음과 같이 함수를 정의하는게 좋아보입니다.
```swift
func setHeight(_ height: CGFloat) {
    ...
}
```

```suggestion
        setHeight(165)
```

```

Reply:
https://github.com/Hongmoon-gak/First-CEO-iOS/pull/8/commits/9d192156adb2d722a40d51350836b485708ad3d0 반영했습니다! 제안 감사합니다.